README.md

```diff
@@ -50,12 +50,11 @@ SAST v9.3 or higher.
 
 Run export with:
 ```
-.\cxsast_exporter --user username --pass password --url http://localhost [--simIdVersion simIdV]
+.\cxsast_exporter --user username --pass password --url http://localhost
 ```
 
  * Replace `username` and `password` with user credentials.
  * Replace `http://localhost` with the url to SAST, if necessary.
```

Comment:
The intention of this part of the README was to provide a basic example on how to make an export, not to document all the options. Please remove this and let users consult `--help` to see all options available.